code/game/machinery/digital_clock.dm

```diff
@@ -153,4 +153,4 @@
 
 	return return_overlays
 
-MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/digital_clock, 28)
+WALL_MOUNT_DIRECTIONAL_HELPERS(/obj/machinery/digital_clock)
```

Comment:
does this actually look ok at all?

Reply:
Might not with arc's fixes; but it inherits being a wallmount from /sign/.

Good thing this wasn't merged though because I just remembered "oh shit I probably missed a button"

Reply:
Arcane knocked this one out of the park for me already; resolved